explore/src/main/scala/explore/HelpBody.scala

```diff
@@ -83,12 +83,17 @@ object HelpBody:
             ExploreStyles.HelpTitle,
             <.h4(ExploreStyles.HelpTitleLabel, "Help"),
             <.div(
-              Button(as = <.a, size = Mini, compact = true, onClick = helpView.set(None))(
-                Icons.Edit
-              )(^.href := editUrl.toString(), ^.target := "_blank"),
-              Button(size = Mini, compact = true, onClick = helpView.set(None))(
-                Icons.Close
-              )
+              <.a(Button(icon = Icons.Edit,
```

Comment:
Cool, I was woing to change these :D
Is the `<.a>` here so you get the link showing?

Reply:
Yes, exactly. We can't do `as` anymore, so this allows the link to show on hover.